### DISCUSSION TITLE
Feedback for Book Features

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#117] [Page Feedback]
Feedback discussion for Book Features: https://csharp-develop.captaincoder.org/book-features

**Comment by cloudthepain** [comment]
I would add an introduction rather than just hoping into it. It starts kind of abruptly.

"This book has many features within it. In this section we will go over... blah blah blah."

Do you want to add a section explaining the difference between activities, challenges, and features of the journal?

**Reply by jcollard** [reply]
Agreed. This stuff was actually on the Intro page before I rewrote it. I have not had a chance to rewrite it just yet.

For now, I am just going to remove it from the left menu.
